sourcemash/frontend/static/js/views/browse_view.js

```diff
@@ -34,22 +34,28 @@ Sourcemash.Views.BrowseView = Backbone.View.extend({
 
   render: function() {
     // Render parent view
-    this.$el.html(this.template({ models: this.collection.models }));
+    this.$el.html(this.template({ models: this.feedTopicViews }));
     $('#typeahead').html(this.typeahead.render().el);
 
     // Render loading view
     this.loadingView = new Sourcemash.Views.LoadingView({loading: this.loading});
     this.$(".loading").html(this.loadingView.render().el);
 
-    // Render item cards
-    var feedCards = [];
-    this.collection.models.forEach(function(feed) {
-        var feedCardView = new Sourcemash.Views.FeedCardView({el: "#feed-card-" + feed.get('id'),
-                                                              model: feed});
-        feedCards.push(feedCardView)
+    feeds = this.collection;
+    var topicViews = [];
```

Comment:
If we add this.close to the render, will we need to maintain this.feedTopicViews at all?